packages/web-components/src/message-bar/message-bar.template.ts

```diff
@@ -0,0 +1,77 @@
+import { ElementViewTemplate, html, when } from '@microsoft/fast-element';
+import type { MessageBar } from './message-bar.js';
+
+const infoIcon = html`
```

Comment:
Should SVGs be baked in, or slotted? Rethinking this now.

Reply:
I don't think we should default these if there are designs given:
1. there are scenarios where you can _not_ have an icon at all, but the bundle will include all 4.
2. any scenario only requires *one* but you will get all 4